packages/Select/src/select-value.scss

```diff
@@ -1,56 +1,71 @@
-@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+@use "~@igloo-ui/tokens/dist/base10/variables" as tokens;
 
 :root {
-  /* Default */
-  --ids-select-value-padding: #{tokens.$space-2} #{tokens.$space-3};
+    /* Default */
+    --ids-select-value-padding: #{tokens.$space-2} #{tokens.$space-3};
 
-  /* Compact **/
-  --ids-select-value-padding-compact: #{tokens.$space-1} #{tokens.$space-3};
+    /* Compact **/
+    --ids-select-value-padding-compact: #{tokens.$space-1} #{tokens.$space-3};
 
-  /* Disabled */
-  --ids-select-value-color-disabled: #{tokens.$grey-500};
+    /* Disabled */
+    --ids-select-value-color-disabled: #{tokens.$grey-500};
 
-  /* Placeholder */
-  --ids-select-value-color-placeholder: #{tokens.$grey-400};
+    /* Placeholder */
+    --ids-select-value-color-placeholder: #{tokens.$grey-400};
 
-  /* Visual Identifier */
-  --ids-select-value-icon-size: #{tokens.$space-4};
-  --ids-select-value-icon-margin-right: #{tokens.$space-2};
+    /* Visual Identifier */
+    --ids-select-value-icon-size: #{tokens.$space-4};
+    --ids-select-value-icon-margin-right: #{tokens.$space-2};
+}
+
+[data-brand="workleap"] {
+    /* Default */
+    --ids-select-value-padding: var(--hop-space-stack-sm) var(--hop-space-inline-md);
+
+    /* Compact **/
+    --ids-select-value-padding-compact: 0.375rem var(--hop-space-inline-md);
```

Comment:
If the padding is 6 because of the border (2 x 1px) this could use the space-inline-md - border-width